18.0 t 49935 jjs

jjscarafia opened the pull request.

jjscarafia committed:
[MIG] public_budget: Migration to 18.0

roboadhoc commented:
[Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/odoo-public-administration/pull/556).

iga-adhoc committed:
[MIG] sipreco_purchase: Migration to 18.0

iga-adhoc committed:
[MIG] public_budget : Fix account assets views

[FIX] public_budget: arreglar states de account.payment.group
1) arreglar states de account.payment.group .
2) las reglas de acceso si o si tienen que tener un grupo.

[WIP] fixes varios

iga-adhoc committed:
[MIG] sipreco_purchase : Fix data demo

iga-adhoc committed:
[MIG] sipreco_subsidy_management: Migration to 18.0

iga-adhoc committed:
[MIG] sipreco_position_helper/: Migration to 18.0

iga-adhoc committed:
[MIG] public_budget_subscription: Migration to 18.0

iga-adhoc committed:
[MIG] sipreco_purchase : Fix states in views and clean warnings

iga-adhoc committed:
[MIG] public_budget_tax_settlement: Migration to 18.0

iga-adhoc committed:
[MIG] sipreco_project: Migration to 18.0

jjscarafia committed:
[WIP] public_budget: payment group to payment

iga-adhoc committed:
[MIG] sipreco_payment_line: Migration to 18.0

iga-adhoc committed:
[MIG] public_budget: Adapt public holidays

jjscarafia committed:
[REM] public_budget: checks usage

Ya no se usan cheques en ninguna de las bases de sipreco, deprecamos todo lo relativo a cheques

jjscarafia committed:
[REM] public_budget_subscription

Solo estaba instalado en CMR y no tenian ni un contrato de compras

jjscarafia committed:
[IMP] public_budget: add post-migration script and fix pre-migration

jjscarafia committed:
[IMP] public_budget: add post-migration script and fix pre-migration

jjscarafia committed:
[FIX] public_budget: chart install
